docs/getting-started-guide/faqs.rst

```diff
@@ -60,17 +61,30 @@ Handling Bounds
 
 How are bounds generated in xCDAT?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-xCDAT generates bounds by using coordinate points as the midpoint between their lower
-and upper bounds.
+For the X, Y, and Z axes, xCDAT generates bounds by using coordinate points as the
+midpoint between their lower and upper bounds.
+
+For the T axis, xCDAT can generate bounds either by 1) time frequency (default method)
+or 2) midpoints.
+
+   1. time frequency: create time bounds as the start and end of each timestep's period
+      using either the inferred or specified time frequency.
+   2. midpoint: create time bounds using time coordinates as the midpoint between their
+      upper and lower bounds.
+
+For more information, visit the documentation for these APIs:
+- https://xcdat.readthedocs.io/en/stable/generated/xarray.Dataset.bounds.add_missing_bounds.html
+- https://xcdat.readthedocs.io/en/stable/generated/xarray.Dataset.bounds.add_time_bounds.html
+- https://xcdat.readthedocs.io/en/stable/generated/xarray.Dataset.bounds.add_bounds.html
 
 Does xCDAT support generating bounds for multiple axis coordinate systems in the same dataset?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 *For example, there are two sets of coordinates called “lat” and “latitude” in the dataset.*
 
 Yes, xCDAT can generate bounds for axis coordinates if they are  “dimension coordinates”
-(`coordinate variables`_ in CF terminology) and have the required CF metadata.
-“Non-dimension coordinates” (`auxiliary coordinate variables`_ in CF terminology) are
-ignored.
+and have the required CF metadata. Dimension coordinates are also considered "index"
+coordinates in Xarray and `coordinate variables`_ in CF terminology. “Non-dimension coordinates”
+(`auxiliary coordinate variables`_ in CF terminology) are ignored.
```

Comment:
Explain why "Non-dimension coordinates" are ignored (because they aren't used as indexes and aren't mapped to the axes of variables)